### ISSUE TITLE
Feature request: Spread and VCF Limit under CC or SYSEX control

### ISSUE THREAD

**Issue by RDPRDP** [image-et-son/p600fw#69]
If not already implemented it would be nice to have Spread (Full CC range, or SYSWEX) and VCF Limit (CC 0-63 OFF, 64-127 ON, or SYSEX) under midi control.

**Comment by matrix12x** [comment]
I was thinking about this and I think this is best handled as sysex. It's not a thing one would change in real time, and it is bets to preserve CCs for real time control.

**Comment by image-et-son** [comment]
> I was thinking about this and I think this is best handled as sysex. It's not a thing one would change in real time, and it is bets to preserve CCs for real time control.

Yes, we're running out of CCs anyway. Maybe it would be possible to implement a general SysEx format for settings.
